packages/remirror__extension-link/src/link-extension.ts

```diff
@@ -371,224 +382,6 @@ export class LinkExtension extends MarkExtension<LinkOptions> {
     };
   }
 
-  /**
-   * Create the paste rules that can transform a pasted link in the document.
-   */
-  createPasteRules(): MarkPasteRule[] {
-    return [
-      {
-        type: 'mark',
-        regexp: this.options.autoLinkRegex,
-        markType: this.type,
-        getAttributes: (url, isReplacement) => ({
-          href: getMatchString(url),
-          auto: !isReplacement,
-        }),
-      },
-    ];
```

Comment:
Paste rule was always applied, even when `this.options.autoLink = false` - is this expected behaviour?

Reply:
I believe it's expected. For example, Google Docs doesn't have auto-linking (you need to parse the space to create a link) but it does have a paste rule for links.